lab_4_fill_words_by_ngrams/main.py

```diff
@@ -28,6 +32,19 @@ def _tokenize(self, text: str) -> tuple[str, ...]:  # type: ignore
         Raises:
             ValueError: In case of inappropriate type input argument or if input argument is empty.
         """
+        if not (isinstance(text, str) and text):
+            raise ValueError("WordProcessor._tokenize: Inappropriate input.")
```

Comment:
`WordProcessor._tokenize` - is in traceback, remove it from the message

Reply:
Ok